snooty.toml

```diff
@@ -14,8 +14,8 @@ avro = "Avro"
 avro-converter="Kafka Connect Avro Converter (Avro Converter)"
 protobuf-converter="Kafka Connect Protobuf Converter"
 json-schema-converter="Kafka Connect JSON Schema Converter"
-connector_version="1.10"
-connector_patch_version="1"
+connector_version="1.11"
+connector_patch_version="0"
```

Comment:
Question: Do we normally put this when there's no patch?

Reply:
We don't list the patch version in a lot of the repos, but in this case the installation page lists the full version using both of these variables. 
If you mean putting 0 when there's no patch (ie 1.11.0 instead of 1.11) then yes that's generally how we show it when we list a full version, although I think it also depends on the repo.